### PR TITLE
Update configuration.md

### PR DESCRIPTION
**Pull request by daixiang0**
Fix [issue](https://github.com/prometheus/alertmanager/issues/2051)
Comment how to set when use smtp servier.

Signed-off-by: Xiang Dai <764524258@qq.com>

### CODE REVIEW

File: content/docs/alerting/configuration.md [hunk 1]
```diff
@@ -70,6 +70,8 @@ global:
   [ smtp_smarthost: <string> ]
   # The default hostname to identify to the SMTP server.
   [ smtp_hello: <string> | default = "localhost" ]
+  # Now alertmanager does not support NTLM authentication, if use SMTP server, please set it as "" or comement it to disable
+  # SMTP authentication.
```
Comment (simonpasquier) [inline content/docs/alerting/configuration.md]: `SMTP username for CRAM-MD5, LOGIN and PLAIN. If empty, Alertmanager doesn't authenticate to the SMTP server.`

Comment (brian-brazil) [reply content/docs/alerting/configuration.md]: To be consistent with the rest, it should be "SMTP Auth using CRAM-MD5, LOGIN and PLAIN."

Comment (daixiang0) [reply content/docs/alerting/configuration.md]: No NTLM? should it be `Alertmanager support SMTP Auth with  CRAM-MD5, LOGIN and PLAIN`?

**Comment by brian-brazil** [comment]
Thanks!

**Comment by daixiang0** [comment]
@brian-brazil still have a question, if use LOGIN auth, it use smtp_from as username and stmp_auth_password as password, right?

**Pull request merged**